src/view/com/util/load-latest/LoadLatestBtn.tsx

```diff
@@ -20,61 +21,62 @@ export function LoadLatestBtn({
   label: string
   showIndicator: boolean
 }) {
-  const pal = usePalette('default')
   const {isDesktop, isTablet, isMobile} = useWebMediaQueries()
   const {fabMinimalShellTransform} = useMinimalShellMode()
 
+  const tokens = useTokens()
+  const styles = useStyles({
```

Comment:
Again didn't really need to do this one here, but few of the components it applies to accept style props (which they could in our ideal world) and it was easy to copy over + part of this draft PR is testing perf so might as well